discord_bots/models.py

```diff
@@ -488,6 +501,28 @@ class SkipMapVote:
     )
 
 
+@mapper_registry.mapped
+@dataclass
+class PersistentView:
+    """
+    Stores persistent view information.
+    To be used when re-initializing views in cogs.
+    """
+
+    __sa_dataclass_metadata_key__ = "sa"
+    __tablename__ = "persistent_view"
+
+    view_id: str = field(
```

Comment:
```suggestion
    id: str = field(
```
for convention lets keep the primary key as `id`